### PR TITLE
feat(metrics): add pvc namespace to volume metrics

### PR DESCRIPTION
**Pull request by antoninferrand**
https://github.com/longhorn/longhorn/issues/7077

### CODE REVIEW

**Review by innobead — DISMISSED** [review]
LGTM

**Comment by innobead** [comment]
@c3y1huang Please help review this.

**Comment by innobead** [comment]
@mantissahz please help review this since Chinya is off today. Thanks.

**Review by mantissahz — APPROVED** [review]
LGTM and Verified.

@antoninferrand just one question.

I got the `namespace` label assigned with `longhorn-system` in default, and after you added this label, the label name would be `exported_namesapce` assigned with `default` if pvc was created in the namespace `default`.
Is this scenario what you expected?

**Comment by mantissahz** [comment]
@Mergifyio backport v1.4.x,v1.5.x

File: metrics_collector/types.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ const (
 	managerLabel         = "manager"
 	backupLabel          = "backup"
 	pvcLabel             = "pvc"
+	namespaceLabel       = "pvc_namespace"
```
Comment (innobead) [inline metrics_collector/types.go:29]: rename `pvcNamespaceLabel`

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by mantissahz** [comment]
>Maybe something like pvc_namespace ? WDYT ?

@antoninferrand That would be great.
I'd changed the label name, could you review it?

File: metrics_collector/types.go (hunk 1 above)
Comment (mantissahz) [reply metrics_collector/types.go]: Modified. Thanks.

**Review by innobead — APPROVED** [review]

**Comment by mantissahz** [comment]
@Mergifyio backport v1.4.x,v1.5.x

**Comment by mantissahz** [comment]
@Mergifyio backport v1.4.x, v1.5.x

**Pull request merged**